packages/cfd/src/Containers/dmt5-trade-modal.tsx

```diff
@@ -101,7 +101,7 @@ const DMT5TradeModal = ({
             <div className='cfd-trade-modal__login-specs'>
                 <div className='cfd-trade-modal__login-specs-item'>
                     <Text className='cfd-trade-modal--paragraph'>{localize('Broker')}</Text>
-                    <SpecBox is_bold is_broker value={'Deriv Holdings (Guernsey) Limited'} />
+                    <SpecBox is_bold is_broker value={'Deriv.com Limited'} />
```

Comment:
Why is this change needed? I cannot see it in the card!

Reply:
We need to update the Broker Code, but unfortunately it is not mentioned in the card  😢

Reply:
And it should not be in the scope of this card so reverting it